mysite/_config.php

```diff
@@ -6,10 +6,13 @@
 global $database;
 
 // find the database name from the environment file
-if(defined('SS_DATABASE_NAME') && SS_DATABASE_NAME) {
+if (defined('SS_DATABASE_NAME') && SS_DATABASE_NAME) {
 	$database = SS_DATABASE_NAME;
-} else {
-	$database = 'SS_cwp';
+} elseif (
+    !defined('SS_DATABASE_CHOOSE_NAME') ||
```

Comment:
can this just be `} elseif (!defined('SS_DATABASE_CHOOSE_NAME') || !SS_DATABASE_CHOOSE_NAME) {`, please?

Reply:
Probability is `1.0`!
> Don't code late at night, kids :<